src/views/Nft/market/Collection/IndividualNFTPage/ForSaleTableCard/ForSaleTableRows.tsx

```diff
@@ -51,6 +61,16 @@ const Row: React.FC<RowProps> = ({ t, nft, bnbBusdPrice, account }) => {
   const [onPresentAdjustPriceModal] = useModal(<SellModal variant="edit" nftToSell={nft} />)
   const { profile, isFetching } = useProfileForAddress(nft.marketData.currentSeller)
   const profileName = profile?.hasRegistered ? profile.profile.username : '-'
+  const sellerProfilePic = profile?.hasRegistered ? profile.profile.nft.images.sm : null
+
+  let sellerProfilePicComponent = <Skeleton width="32px" height="32px" mr="12px" />
+  if (!isFetching) {
+    if (sellerProfilePic) {
+      sellerProfilePicComponent = <Avatar src={`/images/nfts/${sellerProfilePic}`} />
```

Comment:
Akh yea we're really gonna have to unstitch this one before the squad comes out.